src/content/docs/dev/send-tokens/interchain-tokens/developer-guides/link-custom-tokens-deployed-across-multiple-chains-into-interchain-tokens.mdx

```diff
@@ -300,120 +307,158 @@ main().catch((error) => {
+If you see something similar to the following on your console, you have successfully registered your token metadata on AValanche Fuji.
 
-## Deploy a remote token manager on the Polygon testnet
+```bash
+Transaction Hash: 0x026ee7992de2108ecb83f37119ec84ebed371ff724d38e8fd055cbecde5b77e6,
+
+```
+
+### Check the transaction on the BSC testnet scanner
+
+Check the [Fantom testnet scanner](https://testnet.bscscan.com/) to see if you have successfully [registered your token metadata](https://testnet.bscscan.com/tx/0xf860bd8802d046fb0c752c355bc73c87e69f701348420c6cbdf82534b87c4822).
 
-You’ve just successfully deployed a token manager to Fantom, which you are using as your local chain. Now, deploy a token manager remotely to Polygon, which will act as the remote chain in this tutorial. Remember that you can specify any two chains as your local and remote chains.
+### Check the transaction on the Avalanche Fuji testnet scanner
 
-### Estimate gas fees
+Check the [Avalanche Fuji testnet scanner](https://testnet.snowtrace.io/) to see if you have successfully [registered your token metadata](https://testnet.snowtrace.io/tx/0xe3bb64c76637fcd4d9ce9a5bab013fcda95e7040292531230be70daa4f5bd9a1).
 
-In `customInterchainToken.js`, call `estimateGasFee()` from the [AxelarJS SDK](https://www.notion.so/bd238d757a144a069501f0b481488ef3?pvs=21) to estimate the actual cost of deploying your remote Canonical Interchain Token on a remote chain:
+## Register the custom token with the Interchain Token Factory
+
+You've registered your token metadata on BSC and Avalanche Fuji. Now, register the custom token with the Interchain Token Factory on the source chain, which in this example is BSC.
+
+To do that, use the following code snippet: create a function with the name `registerCustomTokenOnBSC`. This function registers an existing ERC20 token under a `tokenId` computed from the provided `salt`.
 
 ```javascript
-//...
+async function registerCustomTokenOnBSC() {
+  // Generate random salt
+  const salt = "0x" + crypto.randomBytes(32).toString("hex");
 
-const api = new AxelarQueryAPI({ environment: Environment.TESTNET });
+  // Get a signer to sign the transaction
+  const signer = await getSigner();
+
+  // Get the interchainTokenFactory contract instance
+  const interchainTokenFactoryContract = await getContractInstance(
+    interchainTokenFactoryContractAddress,
+    interchainTokenFactoryContractABI,
+    signer,
+  );
 
-// Estimate gas costs
-async function gasEstimator() {
-  const gas = await api.estimateGasFee(
-    EvmChain.FANTOM,
-    EvmChain.POLYGON,
-    GasToken.FTM,
-    700000,
-    1.1,
+  // Register token metadata
+  const deployTxData = await interchainTokenFactoryContract.registerCustomToken(
+    salt, // salt
+    bscCustomTokenAddress, // token address
+    MINT_BURN, // token management type
+    "0x510e5EA32386B7C48C4DEEAC80e86859b5e2416C", //  the address of the operator
```

Comment:
we should explain what an operator is at some point.

Reply:
I think we can add this on a separate PR to the main docs and Glossary. I have created an [issue](https://github.com/axelarnetwork/axelar-docs/issues/1298) for it.